src/test_l1_hdmi_cec_driver.c

```diff
@@ -768,15 +777,17 @@ void test_hdmicec_hal_l1_removeLogicalAddress_positive( void )
     result = HdmiCecOpen(&handle);
     //if init is failed no need to proceed further
     UT_ASSERT_EQUAL_FATAL( result, HDMI_CEC_IO_SUCCESS );
-//check need to remove/add fatal
     result = HdmiCecAddLogicalAddress( handle, logicalAddress );
     if (((strncmp(typeString,"sink",UT_KVP_MAX_ELEMENT_SIZE) == 0) && (HDMI_CEC_IO_SUCCESS != result)) || \
```

Comment:
In this scenario, as we are checking for the removal of the logical address, no need to check the return of NOT_SUPPORTED on the addlogical address.  This is similar to the above test where NOT_SUPPORTED is not validated for Addlogical address.

Reply:
Updated